arch/mx.rst

```diff
@@ -104,11 +101,17 @@ The ability to perform analytical SQL queries combined with high volume data ing
 
 An important aspect to consider is that horizontally scaling out your processing power ensures that indexes don’t necessarily become an ingestion bottleneck as your application grows. PostgreSQL has very powerful indexing capabilities and with the ability to scale out you can almost always get the desired read- and write-performance.
 
-Limitations Compared to Citus
------------------------------
+MX Limitations
+--------------
 
-All Citus 7.4 features are supported in Citus MX with the following exceptions:
+Although MX allows direct reading and writing from worker nodes, it doesn't support all commands on workers.
 
-Append-distributed tables currently cannot be made available from workers. They can still be used in the traditional way, with queries going through the coordinator. However, append-distributed tables already allowed you to :ref:`bulk_copy`.
+Limitations and unsupported statements for execution on a worker node:
 
-When performing writes on a hash-distributed table with a bigserial column via the data URL, sequence numbers are no longer monotonic, but instead have the form <16-bit unique node ID><48-bit local sequence number> to ensure uniqueness. The coordinator node always has node ID 0, meaning it will generate sequence numbers as normal. Serial types smaller than bigserial cannot be used in distributed tables.
+* :ref:`DDL <ddl>` commands on distributed tables are not executable from workers, and must be done through the coordinator.
+* Workers may not call :ref:`user_defined_functions` that change Citus metadata.
```

Comment:
This can be done via the coordinator. Can we add a separate section like:
"Supported only via the Coordinator", and add this and the others which work via the coordinator to that? (DDL and Create-table UDF's) 
We can also clarify in that section that since the Coordinator is the authoritative source of the metadata, changes to it only happen via that node, and are propagated to the worker nodes. (Thus the limitation).